tests/foreman/ui_airgun/test_jobinvocation.py

```diff
@@ -0,0 +1,174 @@
+"""Test class for Job Invocation procedure
+
+:Requirement: JobInvocation
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+
+from nailgun import entities
+
+from robottelo.cli.host import Host
+from robottelo.config import settings
+from robottelo.constants import DEFAULT_LOC_ID, DISTRO_RHEL6
+from robottelo.datafactory import gen_string
+from robottelo.decorators import fixture, tier2, tier3
+from robottelo.helpers import add_remote_execution_ssh_key
+from robottelo.vm import VirtualMachine
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@fixture(scope='module')
+def module_subnet(module_org):
+    return entities.Subnet(
+        domain=[entities.Domain(id=1)],
+        gateway=settings.vlan_networking.gateway,
+        ipam='DHCP',
+        location=[entities.Location(id=DEFAULT_LOC_ID)],
+        mask=settings.vlan_networking.netmask,
+        network=settings.vlan_networking.subnet,
+        network_type='IPv4',
+        remote_execution_proxy=[entities.SmartProxy(id=1)],
+        organization=[module_org],
+    ).create()
+
+
+@tier2
+def test_positive_run_default_job_template_by_ip(
+        session, module_org, module_subnet):
+    """Run a job template on a host connected by ip
+
+    :id: 9a90aa9a-00b4-460e-b7e6-250360ee8e4d
+
+    :Setup: Use pre-defined job template.
+
+    :Steps:
+
+        1. Set remote_execution_connect_by_ip on host to true
+        2. Navigate to an individual host and click Run Job
+        3. Select the job and appropriate template
+        4. Run the job
+
+    :expectedresults: Verify the job was successfully ran against the host
+
+    :CaseLevel: Integration
+    """
+    with VirtualMachine(
+        distro=DISTRO_RHEL6,
+        provisioning_server=settings.compute_resources.libvirt_hostname,
+        bridge=settings.vlan_networking.bridge,
+    ) as client:
+        client.install_katello_ca()
+        client.register_contenthost(module_org.label, lce='Library')
+        assert client.subscribed
+        add_remote_execution_ssh_key(client.ip_addr)
+        Host.update({
+            'name': client.hostname,
+            'subnet-id': module_subnet.id,
+        })
+        # connect to host by ip
+        Host.set_parameter({
+            'host': client.hostname,
+            'name': 'remote_execution_connect_by_ip',
+            'value': 'True',
+        })
+        hostname = client.hostname
+        with session:
+            session.organization.select(module_org.name)
```

Comment:
nitpick: redundant line

Reply:
I will be happy to remove that, but only if I find machine to re-test cases as so far only one machine in whole infrastructure sufficed my needs

Reply:
yeah, it seems to hard to find exact moment when qe6.4 machine is not occupied by any activity. Let's remove all these context selections in separate PR